src/main/java/org/broadinstitute/hellbender/utils/dragstr/DragstrUtils.java

```diff
@@ -0,0 +1,283 @@
+package org.broadinstitute.hellbender.utils.dragstr;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+import org.aeonbits.owner.util.Collections;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+public class DragstrUtils {
+
+    public static final String DRAGSTRINFO_KEY = "DRAGstrInfo";
+    public static final String DRAGSTRPARAMS_KEY = "DRAGstrParams";
+
+    public static STRSequenceAnalyzer repeatPeriodAndCounts(final int maxSequenceLength, final int maxPeriod) {
```

Comment:
Looking at this class now. It looks like almost none of these methods are being used anymore? It seems that you intended to rename and push all of this code into the DragstrReferenceAnalyzer class which seems tobe doing all of the work this class used to do about computing the str presence for a given bit of sequence. Perhaps you mean to delete this class and lost it in the rebase?

Reply:
Will see to it, refactor or removal.

Reply:
Done